packages/metatable/src/utils.ts

```diff
@@ -103,9 +103,13 @@ export const toMetaFilters = <TColumns extends Columns<TTypes>, TTypes>(columns:
   );
 };
 
-type Options = { submitLabel?: string; label?: string };
-export const getStringFilter = (path: string[], value?: IStringInput['filters'], options?: Options): MetaField[] =>
-  [{
+type Options = { value?: IStringInput['filters'], label?: string, submit?: MetaField };
+
+const exist = <T>(value: T): boolean => !!value;
+const filter = <T>(array: T[]): T[] => array.filter(exist);
```

Comment:
Tohle nedává smysl. Jestli je to pole pokaždý typu T tak neni potřeba filtrovat na existující, protože všechny prvky budou existovat.  
Smysl dává to otypovat takhle: 

```
<T>(array: Array<T | undefined>): T[] => array.filter(exist);
```

taky mi přijde zbytečný dělit to do dvou funkcí když to exist nikde jinde nepoužíváš:

```
<T>(array: Array<T | undefined>): T[] => array.filter(value => !!value);

```

Reply:
ok, je to tam